07.bowling_object/lib/frame.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  attr_reader :shots
+
+  def initialize(*roll)
+    @shots = roll
+  end
+
+  def total_score
+    shots.sum(&:score)
+  end
+
+  def scores
+    shots.map(&:score)
+  end
+
+  def strike?
+    shots[0].score == STRIKE
```

Comment:
```suggestion
    shots[0].strike?
```

Shotクラスに「君、ストライク？」って尋ねる方がオブジェクト指向らしいですね。